### ISSUE TITLE
DocBlockBase error

### ISSUE THREAD

**Issue by YasKur** [pocka/storybook-addon-designs#109]
**Affected design types**

- `figma`

**Describe the bug**
Error 1
```
blocks.js:49 Uncaught TypeError: Cannot read property 'resetwrapper' of undefined
at DocBlockBase (blocks.js:49)
at renderWithHooks (react-dom.development.js:14803)
at mountIndeterminateComponent (react-dom.development.js:17482)
at beginWork (react-dom.development.js:18596)
at HTMLUnknownElement.callCallback (react-dom.development.js:188)
at Object.invokeGuardedCallbackDev (react-dom.development.js:237)
at invokeGuardedCallback (react-dom.development.js:292)
at beginWork$1 (react-dom.development.js:23203)
at performUnitOfWork (react-dom.development.js:22154)
at workLoopSync (react-dom.development.js:22130)
```
Error 2: 
The above error occurred in the <DocBlockBase> component:
    in DocBlockBase (created by Figma)
    in Figma (at figma-doc-page/index.jsx:13)
    in page
    in div (created by Context.Consumer)
    in Styled(div) (created by DocsContainer)
    in div (created by Context.Consumer)
    in Styled(div) (created by DocsContainer)
    in MDXProvider (created by DocsContainer)
    in ThemeProvider (created by DocsContainer)
    in SourceContainer (created by DocsContainer)
    in DocsContainer

**To Reproduce**
1. Run storybook.
2. Go to page component, click on tab Docs and see this error in console. 

In version 6.0.0-alpha.3 all work correct, but in current version 6.0.0 u can see this errors.
```
export default {
  title: 'Example'
  component: Example,
  decorators: [withDesign],
  parameters: {
    docs: {
      page: () => <Figma url={url} />
    },
  },
};
```
**Versions**
└─┬ @storybook/addon-docs@6.0.27
  └─┬ @storybook/core@6.0.27
    └── @storybook/ui@6.0.27 
"storybook-addon-designs": "6.0.0";

**Comment by pocka** [comment]
It seems [this](https://github.com/pocka/storybook-addon-designs/blob/a79f847bb55297cb471e9f44f787ded96326a10d/packages/storybook-addon-designs/src/blocks.tsx#L116) causes the error but I have no idea why [that import](https://github.com/pocka/storybook-addon-designs/blob/a79f847bb55297cb471e9f44f787ded96326a10d/packages/storybook-addon-designs/src/blocks.tsx#L5) would be undefined...
I introduced changes to the file in https://github.com/pocka/storybook-addon-designs/commit/3bf4551cb89129c15cdcd65a1b7ad0a1365d2849. I believe it's compatible even with Storybook v6.0.x.

@YasKur 
Can you share a reproduction repository or sandbox? Also, the version of `@storybook/components` you're using would be helpful.

**Comment by YasKur** [comment]
@pocka components storybook
```
"@storybook/addon-actions": "6.0.27",
"@storybook/addon-backgrounds": "6.0.27",
"@storybook/addon-docs": "6.0.27",
"@storybook/addon-links": "6.0.27",
"@storybook/addons": "6.0.27",
"@storybook/preset-create-react-app": "3.1.4",
"@storybook/react": "6.0.27",
```
error with - "storybook-addon-designs": "6.0.0", but "storybook-addon-designs": "6.0.0-alpha.3" work good :)

**Comment by pocka** [comment]
The patch is available in [v6.0.1](https://github.com/pocka/storybook-addon-designs/releases/tag/v6.0.1) and [v5.4.6](https://github.com/pocka/storybook-addon-designs/releases/tag/v5.4.6).
